nectar/src/maker/strategy.rs

```diff
@@ -116,6 +122,10 @@ impl AllIn {
         mid_market_rate: Rate,
     ) -> Result<BtcDaiOrderForm> {
         if quote_balance <= self.dai_reserved_funds {
+            sentry::capture_message(
+                "DAI balance too low to create order",
```

Comment:
Reading that, it confirm my opinion that we should hook sentry to tracing warn and error entries.

Please do it, then review warn and error tracing entries and confirm they are relevant.

Reply:
If we get a feature request from someone else running the software we can do it, otherwise I would just do what I need at the moment, see my comment below :)